services/src/test/scala/cromwell/pipeline/service/ProjectFileServiceTest.scala

```diff
@@ -79,22 +78,28 @@ class ProjectFileServiceTest extends AsyncWordSpec with Matchers with MockitoSug
       val projectId = project.projectId
       val projectFileContent = ProjectFileContent("File content")
       val projectFile = ProjectFile(Paths.get("test.txt"), projectFileContent)
+      val version = Some(TestProjectUtils.getDummyPipeLineVersion())
+      val userId = UserId.random
 
       "return success message for request" taggedAs Service in {
+        when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future(project))
```

Comment:
```suggestion
        when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future.successful(project))
```
Same for other places below

Reply:
Fixed.